README.md

```diff
@@ -143,9 +157,9 @@ and `selectedSlide` are specified, both will be applied with the latter overridi
 
 Example:
 
-```
+```jsx
 <Carousel
-  ...
+  // ...
```

Comment:
This isn't technically valid jsx either.  The intention here is just to show additional props would still be necessary here but highlighting the portion being written about. So this already can't be copy-pasted anywhere so I'm not too worried about it being valid jsx and was fine how it was written before

Reply:
I think this comment is between props is valid jsx, since not even eslint complains about it nor `react-scripts`. I think is better than not working.

Reply:
>  I think is better than not working.

Is it? it's incomplete, you can't copy and paste it as is. If someone tries, should it just look like it works or fail out hard before you even get there.  the intention is to show that it is incomplete.